src/exo/platforms/gemmini.py

```diff
@@ -544,15 +544,15 @@ def make_ld_i8_block(name, ld_id, p=ld_i8_block, stride_val=None):
     "ld_i8_block_id1_v2", 1, stride_val="stride(src,0)"
 )
 ld_i8_block_id1_s2_v2 = make_ld_i8_block(
-    "ld_i8_block_id1_s2_v2", 1, stride_val="stride(src,2)"
+    "ld_i8_block_id1_s2_v2", 1, stride_val="stride(src,0)"
```

Comment:
Do these really not make it into the golden outputs of _any_ test?

Reply:
Yeah!